config/puma.rb

```diff
@@ -82,3 +82,6 @@
 # - Phased Restarts are only supported in cluster mode with multiple workers (i.e., not in development).
 # - The Puma binary won't be upgraded on phased restarts, but since we have the unattended-upgrades, this is not a major issue.
 # - See https://github.com/casperisfine/puma/blob/master/docs/restart.md.
+
+# Run Solid Queue's supervisor
+plugin :solid_queue
```

Comment:
There is one issue with the puma plugin requiring further investigations: https://github.com/rails/solid_queue/issues/241

Reply:
Further, there is another warning issues with the introduction of the solid_queue puma plugin (which is new): https://github.com/rails/solid_queue/issues/243

Reply:
Based on these issues (and the logging experience), I would recommend not to use the puma plugin. Given the patch supplied above, one can simply start the supervisor through `rails solid_queue:start` or (ideally) through the IDE. 

<details>
<summary>Below are three screenshots showing my configuration</summary>

**Run configuration for Rake task:**
<img width="1137" alt="Bildschirmfoto 2024-06-18 um 23 50 37" src="https://github.com/openHPI/codeharbor/assets/7300329/292b8adc-bb95-4353-ae4b-ab15db0cd7e2">

**Configure a custom log file:**
<img width="716" alt="Bildschirmfoto 2024-06-18 um 23 51 53" src="https://github.com/openHPI/codeharbor/assets/7300329/02a4836a-2847-476a-9540-bc14c8f3f1fd">

**Optionally, use compound to start multiple services together:**
<img width="1137" alt="Bildschirmfoto 2024-06-18 um 23 52 07" src="https://github.com/openHPI/codeharbor/assets/7300329/4baf9102-8d61-47cf-bb82-578c24bb913e">

</details>

With this configuration applied, Solid Queue is debuggable through the IDE (and so are jobs), and logs are split into two tabs:

<img width="305" alt="Bildschirmfoto 2024-06-18 um 23 55 20" src="https://github.com/openHPI/codeharbor/assets/7300329/c05c56db-1ef7-44e9-9e93-0dd2fc0dd632">

The first one contains the most important information, while the second one reading the file log additionally contains UUIDs for the job and more verbose SQL statements.

Given this change, we also need to ensure that the project's README is up-to-date and inform fellow devs about the change via email (ideally with a proposal for their IDE).

Reply:
Okay, I removed the puma plugin. We still have to update the README though.